modules/fedimint-dummy-server/Cargo.toml

```diff
@@ -21,6 +21,7 @@ async-trait = { workspace = true }
 erased-serde = { workspace = true }
 fedimint-core = { workspace = true }
 fedimint-dummy-common = { workspace = true }
+fedimint-server = { workspace = true }
```

Comment:
This is actually a significant downgrade because `fedimint-server` depends on a lot of stuff and is quite heavy.

Had we moved server trait to `fedimint-server-core`, then we would be only adding very lightweight dependency instead.

Reply:
@elsirion I see `fedimint-server-core`, do these still need updating? I hope. :)

Reply:
Right, forgot that, fixed for `dummy`, `empty` and `unknown`. Unfortunately any non-trivial server module still depends on `fedimint-server` for DKG anyway.